ui-tests/src/test/resources/features/integrations/dynamodb.feature

```diff
@@ -0,0 +1,183 @@
+# @sustainer: tplevko@redhat.com
+
+@ui
+@datamapper
+@dynamodb
+@integrations-dynamodb
+Feature: Dynamodb connector
+
+  Background: Clean application state
+    Given clean application state
+    Given log into the Syndesis
```

Comment:
Two Given steps. The second one can be And as well for more readability.

Reply:
should be OK now